app/frontend/src/search/index.js

```diff
@@ -3,7 +3,8 @@ import 'regenerator-runtime/runtime';
 import '../polyfill/classlist.polyfill';
 
 import {
-  connectSearchBox, connectAutocomplete, connectHits, connectSortBy, connectMenu, connectStats, connectPagination,
+  connectSearchBox, connectHits, connectSortBy, connectMenu, connectStats, connectPagination,
+  connectSearchBox, connectHits, connectSortBy, connectMenu, connectStats,
```

Comment:
duplicated imports duplicated imports

Reply:
i did have a rebase conflict here and ended up with 2 but i removed and that is case in file chnages? ill resolve this